From 69647e5d393a52ed3892eccc172ee750d6aaa45d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 30 Dec 2022 21:08:25 -0800
Subject: [PATCH] configure: Do not add -D_FILE_OFFSET_BITS to CFLAGS

AC_SYS_LARGEFILE macro is in use and this will add a definition for
_FILE_OFFSET_BITS in generated config.h which is already included as
first include file in fatresize.c hence its not required to be added via
CFLAGS, this also fixes a case when -D_FILE_OFFSET_BITS=64 is passed via
CC from environment, where the autoconf macros set
ac_cv_sys_file_offset_bits=no and that means we will have
-D_FILE_OFFSET_BITS=no added to CFLAGS which messes up builds.

Upstream-Status: Submitted [https://salsa.debian.org/parted-team/fatresize/-/merge_requests/3]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d6e6cb2..4dcec28 100644
--- a/configure.ac
+++ b/configure.ac
@@ -29,7 +29,6 @@ AC_CHECK_FUNCS([memset strtoll])
 
 # Check for LFS
 AC_SYS_LARGEFILE
-CFLAGS="$CFLAGS -D_FILE_OFFSET_BITS=${ac_cv_sys_file_offset_bits}"
 
 dnl libparted
 # hack
-- 
2.39.0

